documentation/book/proc-creating-new-image-from-base.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// assembly-using-kafka-connect-with-plugins.adoc
+
+include::common/attributes.adoc[]
+
+[id='creating-new-image-from-base-{context}']
+= Create a new image based on our base image
+
+{ProductName} provides its own Docker image for running Kafka Connect, which can be found on {DockerRepository} as
+`{DockerKafkaConnect}`.
+This image can be used as a base image for building a new custom image with additional plugins.
+
+The following procedure describes the process for creating such a custom image.
+
+.Procedure
+
+. Create a new `Dockerfile` using `{DockerKafkaConnect}` as the base image
++
+[source,Dockerfile,subs="attributes"]
+----
+FROM {DockerKafkaConnect}
+USER root:root
+COPY ./my-plugin/ /opt/kafka/plugins/
```

Comment:
Is 'my-plugin' a replaceable?

Reply:
Yes. That is where the users should put the location of the plugin(s) which should be added to the Docker image.